packages/cli/lib/validation.js

```diff
@@ -26,14 +26,19 @@ const fs = require('fs');
 const path = require('path');
 const { EXIT_CODES } = require('./enums/exitCodes');
 
-async function loadAndValidateOptions(options) {
+async function loadAndValidateOptions(options, shouldValidateAccount = true) {
   setLogLevel(options);
   logDebugInfo(options);
   const { config: configPath } = options;
   loadConfig(configPath, options);
   checkAndWarnGitInclusion(getConfigPath());
 
-  if (!(validateConfig() && (await validateAccount(options)))) {
+  let validAccount = true;
+  if (shouldValidateAccount) {
```

Comment:
Added an optional account validation prop to bypass the `validateAccount()` check if needed. This is because if the default account in the config file does not exist in the portals list (edge case), it logs and error and exits, leaving the user no option to set a new default account using commands. 

Let me know if you'd like to handle this differently, I had extracted it at first but figure adding one prop here would be cleaner